src/directives/vRotate.ts

```diff
@@ -0,0 +1,61 @@
+import { shallowRef } from 'vue'
+import { Quaternion, Vector3 } from 'three'
+import type { TresObject } from '../types'
+import { useLogger, useRenderLoop } from '../composables'
+
+const { logWarning } = useLogger()
+
+export const vRotate = {
+  mounted: (
+    el: TresObject,
+    binding: {
+      arg: 'x' | 'y' | 'z'
+      value: number
+      modifiers: Partial<{ x: boolean; y: boolean; z: boolean }>
+    },
+  ) => {
+    if (el.isCamera) {
+      logWarning(`Rotate the ${el.type} is not a good idea`)
+      return
+    }
+    const speed = binding.value ?? 0.01
+    const defaultQuaternion = new Quaternion()
+    const quaternionX = shallowRef(
+      binding.modifiers.x || binding.arg === 'x'
+        ? new Quaternion().setFromAxisAngle(new Vector3(1, 0, 0), speed)
+        : defaultQuaternion,
+    )
+    const quaternionY = shallowRef(
+      binding.modifiers.y || binding.arg === 'y'
+        ? new Quaternion().setFromAxisAngle(new Vector3(0, 1, 0), speed)
+        : defaultQuaternion,
+    )
+    const quaternionZ = shallowRef(
+      binding.modifiers.z || binding.arg === 'z'
+        ? new Quaternion().setFromAxisAngle(new Vector3(0, 0, 1), speed)
+        : defaultQuaternion,
+    )
```

Comment:
(Non-blocking)

You could get by with a single quaternion here, rather than using a quaternion per axis. Something like this pseudocode:

```
new Quaternion().setFromAxisAngle(new Vector3(x, y, z).normalize(), speed)
```

That would have the positive (imho) side effect of maintaining a consistent rotational speed, regardless of arguments, for a given frame rate. 

E.g., as it stands, if the user passes `x` and `y`, the object rotates around the x-axis at `speedPerFrame` and around the y-axis at `speedPerFrame`. With normalization, the sum of all rotations will be a consistent `speedPerFrame`.

Reply:
Hi I tried this, but I get weird behaviors, the cube rotate, but started to grow in every rotation...
So I decided to go with the other approach

Reply:
Like this?

https://github.com/Tresjs/tres/assets/20469369/3f6caa15-343c-430c-a758-3baf91ceec95

If so, it might be due to a non-normalized `Quaternion` or non-normalized axis `Vector3`. Here's the code that produced clip, for example:

```
useRenderLoop().onLoop(({elapsed}) => {
    if (boxRef.value) {
      boxRef.value.value.applyQuaternion(new Quaternion().setFromAxisAngle(new Vector3(1,1,1), 0.1))
    }
})
```

And here's the normalized version:

https://github.com/Tresjs/tres/assets/20469369/921a1163-2592-4320-8071-cf562dcec101


```
useRenderLoop().onLoop(({elapsed}) => {
    if (boxRef.value) {
      boxRef.value.value.applyQuaternion(new Quaternion().setFromAxisAngle(new Vector3(1,1,1).normalize(), 0.1))
    }
})
```

Reply:
Ooohhh yess exactly like that, Thanks.

It was because is not normalize, thanks my friend ❤️